decentraland/redux/reducers.js

```diff
@@ -87,17 +87,31 @@ const landToBuy = createReducer(null, {
 //
 const prependLandForSaleToList = (state, action) => {
   const { landForSale } = action;
+  const { id } = landForSale;
+  let { list: assetsForSale } = state;
+  console.info("previous for sale list length", assetsForSale.length);
+  console.info("prepend landForSale id", id);
+
+  // TODO: Consider having this function ensure that landForSale
+  // isn't already in the list before adding it
   return { ...state, list: [landForSale, ...state.list] };
 };
 
 const appendLandForSaleToList = (state, action) => {
   const { landForSale } = action;
+  const { id } = landForSale;
+  // let { list: assetsForSale } = state;
+  // console.info("previous for sale list length", assetsForSale.length);
+  console.info("append landForSale id", id);
   return { ...state, list: [...state.list, landForSale] };
 };
 
 const removeLandForSale = (state, action) => {
   const { landForSale } = action;
+  const { id } = landForSale;
+  console.info("remove landForSale id", id);
   let { list: assetsForSale } = state;
```

Comment:
@marcelomorgado Any reason not to convert this to a `const`?

Reply:
No, no reason.

Reply:
Updated this in another feature branch. Closing this.